src/coredump/coredump.c

```diff
@@ -1072,7 +1078,9 @@ static int process_kernel(int argc, char* argv[]) {
         context[CONTEXT_UID] = argv[CONTEXT_UID + 1];
         context[CONTEXT_GID] = argv[CONTEXT_GID + 1];
         context[CONTEXT_SIGNAL] = argv[CONTEXT_SIGNAL + 1];
-        context[CONTEXT_TIMESTAMP] = argv[CONTEXT_TIMESTAMP + 1];
+        strncat(timestamp, argv[CONTEXT_TIMESTAMP + 1], 10);
+        strncat(timestamp, "000000", 6);
+        context[CONTEXT_TIMESTAMP] = timestamp;
```

Comment:
@keszybz , Sorry the xattr issue is not due to this patch.   The coredumps are stored in tmpfs.  Looks like that tmpfs is not supporting xattrs.  I tried setting xattrs on normal files, even there it is not getting set.